canopy/app/saplings/userSaplings

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "displayName": "Vanilla",
+    "namespace": "vanilla",
+    "runtimeFiles": [
+      "0.0.0.0:8675/vanilla.js"
+    ],
+    "styleFiles": [],
+    "workerFiles": [],
+    "icon": "https://via.placeholder.com/48"
```

Comment:
is it possible to have this stuff self-hosted out of a local directory?

Reply:
The icon specifically? In practice it would come from wherever the sapling is being hosted.

Reply:
Per in-person conversation this was converted from an existing json example. I'm ok deferring this for a future story which does a pass on self hosting as many assets as are reasonable (gameroom vue icons, this, etc.)